completion/git-machete.completion.bash

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 
 _git_machete() {
-  local cmds="add advance anno d delete-unmanaged diff discover e edit file fork-point g go help is-managed l list log reapply s show slide-out squash status traverse update version"
+  local cmds="add advance anno d delete-unmanaged diff discover e edit file fork-point g go help hub is-managed l list log reapply s show slide-out squash status traverse update version"
```

Comment:
TODO: any change of the subcommand name is to be reflected in the completion scripts